admin/class-admin.php

```diff
@@ -39,8 +36,11 @@ function __construct() {
 			add_action( 'delete_category', array( $this, 'schedule_rewrite_flush' ) );
 		}
 
-		$this->page_gsc = new WPSEO_GSC();
-		$this->dashboard_widget = new Yoast_Dashboard_Widget();
+		$this->admin_features = array(
+			'gsc'              => new WPSEO_GSC(),
+			'primary_category' => new WPSEO_Primary_Term_Admin(),
+			'dashboard_widget' => new Yoast_Dashboard_Widget(),
```

Comment:
I see  `dashboard_widget` is moved to this array, be it's not used further in this class. So why storing this in an array?